lib/tasks/loader.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# Loaders can bulk load data
+namespace :loader do
+  # csv loader can bulk load SearchEvents and Terms
+  #
+  # @note the csv should be formated as `term phrase`, `timestamp`. A dataclip is available that can export in this
+  #   format.
+  # @example
+  # bin/rails loader:csv['local_path_to_file.csv', 'some-source-to-use-for-all-loaded-records']
+  #
+  # @param path [String] local file path to a CSV file to load
+  # @param source [String] source name to load the data under
+  desc 'Load search_events from csv'
+  task :csv, %i[path source] => :environment do |_task, args|
```

Comment:
I think this task should be renamed `search_events` to allow for additional loaders. Alternatively, we could rename the namespace to `search_events` and then the task to `csv_loader`.

The additional loader I am thinking about now is for loading journal information so I'm leaning more towards keeping things separate and having the namespace be the "model" they are loading into and the tasks be the "format_loader" they are loading from.

Reply:
I'm going to assume that as nobody assigned this PR to themselves yet I can go ahead and make this change.